EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFUnpackerInputs.cc

```diff
@@ -36,7 +36,7 @@ namespace l1t
 				return (value == 0);
 		}
 
-		bool unpacking(const Block& block, UnpackerCollections *coll, std::map<int, qualityHits>& linkAndQual_, const bool& isNewFw)
+		bool unpacking(const Block& block, UnpackerCollections *coll, qualityHits& linkAndQual_, const bool& isNewFw)
```

Comment:
@rekovic - in a followup PR, please change the name of linkAndQual_ to be comply with the usual naming convention (blah_ is for class member data)